docs/assistant/security-assistant.asciidoc

```diff
@@ -52,15 +52,39 @@ NOTE: Elastic can automatically anonymize event data that you provide to AI Assi
 [[set-up-ai-assistant]]
 == Set up AI Assistant
 
-You must complete these steps before you can use AI Assistant:
+You must create a generative AI connector before you can use AI Assistant. 
 
-. Create an API key with your AI provider to authenticate requests from AI Assistant. You'll use this in the next step. Refer to the provider's documentation for generating API keys:
-+
-* https://platform.openai.com/docs/api-reference[OpenAI]
-* https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference[Azure OpenAI Service]
+. Open AI Assistant *Cmd + ;* (or *Ctrl + ;* on Windows), and click **Connector** *->* *Add new Connector**
+. Select either *Amazon Bedrock* or *OpenAI*. 
+. Enter the authentication details required for your chosen connector type, then click *Save*.
+
+For OpenAI and Azure OpenAI Service, you need to provide an API key. For Amazon Bedrock, you need to provide an access key and secret for an IAM user with at least the following permissions:
+
+.Expand permissions JSON
```

Comment:
IMO this example is short enough that it doesn't need to be collapsible, but it's also fine like this. If you keep it collapsible, maybe instruct the reader a bit more directly:
```suggestion
.Click to expand permissions JSON
```